likecoin/assets/js/admin/likecoin_metabox.js

```diff
@@ -75,8 +75,14 @@ function onSubmitToISCN(e) {
 }
 
 (() => {
+  const submitISCNHash = window.location.hash;
   const refreshBtn = document.getElementById('lcPublishRefreshBtn');
   const submitISCNBtn = document.getElementById('lcISCNPublishBtn');
+  if (submitISCNHash === '#likecoin_submit_iscn') {
+    setTimeout(() => {
+      window.scrollTo(0, document.querySelector('#poststuff').scrollHeight);
```

Comment:
When is `#poststuff` defined? Can we define `#likecoin_submit_iscn` in our metabox instead?

Reply:
I got that by using developer tool. But yes, I think the later is more specific!